common-treble.mk

```diff
@@ -138,8 +138,16 @@ PRODUCT_PACKAGES += \
 
 # Only define bootctrl HAL availability on AB platforms:
 ifeq ($(AB_OTA_UPDATER),true)
+# Define HAL 1.0 for platforms reading it from GPT
+ifneq ($(filter lena, $(SOMC_PLATFORM)),)
+PRODUCT_PACKAGES += \
+    android.hardware.boot@1.0-impl \
+    android.hardware.boot@1.0-impl.recovery \
+    android.hardware.boot@1.0-service
```

Comment:
I remember that the 1.1 service can start the 1.1 and 1.0 passthrough. Perhaps we only need to select on the -impl for Lena.

Then you can remove the vintf workaround as well.

Reply:
https://android.googlesource.com/platform/hardware/interfaces/+/refs/tags/android-11.0.0_r46/boot/1.1/default/service.cpp#26 fwiw.